CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1142,6 +1168,49 @@ for i = 1, HOTBAR_SLOTS do
 	end
 end
 
+-- Up arrow to open the inventory
+OpenInventoryButton = NewGui('ImageButton', 'OpenInventory')
+OpenInventoryButton.Size = UDim2.new(0, 44, 0, 40)
+OpenInventoryButton.Position = UDim2.new(0.5, -22, 1, -120)
+OpenInventoryButton.Image = "rbxasset://textures/ui/Backpack/ScrollUpArrow.png";
+OpenInventoryButton.MouseButton1Click:connect(function()
+	BackpackScript.OpenClose()
+end)
+
+CloseInventoryButton = NewGui('ImageButton', 'CloseInventory')
+CloseInventoryButton.Size = UDim2.new(0, 40, 0, 40)
+CloseInventoryButton.Position = UDim2.new(0, 0, 0, -50)
+CloseInventoryButton.Image = "rbxasset://textures/ui/Keyboard/close_button_background.png";
+CloseInventoryButton.Visible = true
+CloseInventoryButton.MouseButton1Click:connect(function()
+	if InventoryFrame.Visible then
+		BackpackScript.OpenClose()
+	end
+end)
+do
+	CloseInventoryButtonSelectionObject = NewGui('ImageLabel', 'Selection')
+	CloseInventoryButtonSelectionObject.Size = UDim2.new(0.9,0,0.9,0);
+	CloseInventoryButtonSelectionObject.Position = UDim2.new(0.05,0,0.05,0);
+	CloseInventoryButtonSelectionObject.Image = "rbxasset://textures/ui/Keyboard/close_button_selection.png";
+	CloseInventoryButton.SelectionImageObject = CloseInventoryButtonSelectionObject
+
+	CloseInventoryButtonIcon = NewGui('ImageLabel', 'Icon')
+	CloseInventoryButtonIcon.Size = UDim2.new(0.5,0,0.5,0);
+	CloseInventoryButtonIcon.Position = UDim2.new(0.25,0,0.25,0);
+	CloseInventoryButtonIcon.Image = "rbxasset://textures/ui/Keyboard/close_button_icon.png";
+	CloseInventoryButtonIcon.Parent = CloseInventoryButton;
```

Comment:
Are both CloseInventoryButtonSelectionObject and CloseInventoryButtonIcon intended to be global?